src/Gestures/Gesture.vala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 elementary, Inc (https://elementary.io)
+ *           2021 José Expósito <jose.exposito89@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace Gala {
+    public enum GestureType {
+        NOT_SUPPORTED = 0,
+        SWIPE = 1,
+        PINCH = 2,
+        SCROLL,
+    }
```

Comment:
Could we use https://valadoc.org/gdk-3.0/Gdk.EventType.html for this?

Reply:
True. I was keeping those for compatibility with Touchégg, but it makes more sense to transform into common types in the Touchégg backend.